makefiles/nrf_modem_fw.inc.mk

```diff
@@ -0,0 +1,14 @@
+#include download firmware information from nrf_modem pkg
```

Comment:
I think we should make this independent of the package. I'd move it to its dedicated directory in `dist/tools`, as it really does not need the package, it downloads the firmware independently. What do you think?